tests/src/Kernel/Access/WikiTest.php

```diff
@@ -204,15 +225,6 @@ public function entityOperationAccessProvider() {
         ],
       ],
       [
-        // Non-members should have the right to edit any group content.
```

Comment:
There's isn't really `wiki` feature - the name of the file can change. It's just about checking different scenarios of permissions. So `wiki` refers here to the fact that a non-member can edit existing group content.